src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1044,6 +1044,12 @@
   - name: Create a web app with a NodeJS 6.2 runtime and deployed from a local git repository.
     text: >
         az webapp create -g MyResourceGroup -p MyPlan -n MyUniqueAppName --runtime "node|6.2" --deployment-local-git
+  - name: Create a web app with an image from DockerHub.
+    text: >
+        az webapp create -g MyResourceGroup -p MyPlan -n MyUniqueAppName -i nginx
+  - name: Create a web app with an image from a private Azure Container Registry.
```

Comment:
thanks for adding this - can you also add an example for Private Docker Registry with username & pwd parameters to use?